src/status_im/translations/fr.cljs

```diff
@@ -256,59 +236,133 @@
+   :transactions-unsigned                 "Non-Signé"
+   :transactions-history-empty            "Aucune transaction dans votre historique pour le moment"
+   :transactions-unsigned-empty           "Vous n'avez aucune transaction non signée"
+   :transactions-filter-title             "Filtrer l'historique"
+   :transactions-filter-tokens            "Jetons"
+   :transactions-filter-type              "Type"
+   :transactions-filter-select-all        "Tout sélectionner"
+   :view-transaction-details              "Voir le détail de la transaction"
+   :transaction-description               "Veuillez attendre au moins 12 confirmations pour être sûr que votre transaction a bien été définitivement validée"
+   :transaction-sent                      "Transaction envoyée"
+   :transaction-moved-text                "La transaction va rester dans la liste 'Non-signée' pendant 5 mins"
+   :transaction-moved-title               "Transaction déplacée"
+   :sign-later-title                      "Signer la transaction ultérieurement?"
+   :sign-later-text                       "Vérifier l'historique de la transaction pour la signer"
+   :not-applicable                        "Ne s'applique pas aux transactions non signée"
+
+   ;; Wallet Send
+   :wallet-choose-recipient               "Choisir le destinataire"
+   :wallet-choose-from-contacts           "Choisir à partir des Contacts"
+   :wallet-address-from-clipboard         "Utiliser l'adresse du presse-papiers"
+   :wallet-invalid-address                "Adresse invalide: \n {{data}}"
+   :wallet-browse-photos                  "Parcourir les Photos"
+   :validation-amount-invalid-number      "Le montant n'est pas valide"
+   :validation-amount-is-too-precise      "Le montant est trop précis. La plus petite unité que vous pouvez saisir est 1 Wei (1x10^-18 ETH)"
+
+
+
+   ;; network settings
+   :new-network                           "Nouveau réseau"
+   :add-network                           "Ajouter un réseau"
+   :add-new-network                       "Ajouter un nouveau réseau"
+   :existing-networks                     "Réseaux existants"
+   :add-json-file                         "Ajouter un fichier JSON"
+   :paste-json-as-text                    "Coller JSON en text"
+   :paste-json                            "Coller JSON"
+   :specify-rpc-url                       "Préciser une URL RPC"
+   :edit-network-config                   "Modifier la configuration réseau"
+   :connected                             "Connecté"
+   :process-json                          "Interprétez le JSON"
+   :error-processing-json                 "Erreur survenue lors de l'interprétation du JSON"
+   :rpc-url                               "URL RPC"
+   :remove-network                        "Retirer le réseau"
+   :network-settings                      "Paramètres réseau"
+   :edit-network-warning                  "Soyez prudent, la modification des données réseau pourrait déactiver ce réseau pour vous"
```

Comment:
"Soyez prudent, la modification des données réseau pourrait rendre ce réseau inutilisable pour vous"

Reply:
yep correcting it.